### PR TITLE
Fix some issues blocking WebOS 1.2 / 2.0

### PR DESCRIPTION
**Pull request by JustAMan**
**Changes**

* Fix silent app crash on WebOS 1.2 when app requested notification permission (this code is more correct anyway, and permission is granted on WebOS by default)
* Fix WebOS detection by user agent (typo - `big O` changed to `zero`).
* Create home tabs element properly using `document.createElement()` because instead the polyfill used would add custom class later (based on document mutation), and we need custom methods in this execution context. This is a more correct thing to do in any case, it's just in modern browsers element is _usually_ parsed into custom element immediately (but WWW standards do not require so).
* Require `fetch` explicitly and call Babel on some more modules within `node_modules` so WebOS doesn't choke on them.

**Issues**

No ready-made issues, but this being applied to latest master allows me to open my test JF instance in our jf-webos app in WebOS 1.2 and WebOS 2.0 (in emulators). Still not completely functional, but it's a start.

### CODE REVIEW

File: src/scripts/site.js [hunk 1]
```diff
@@ -709,6 +709,10 @@ var AppInfo = {};
             onError: onRequireJsError
         });
 
+        require(["fetch"], function (fetch) {
+            console.debug("fetch is here", fetch);
```
Comment (MrTimscampi) [inline src/scripts/site.js:713]: Why the debug log?

**Comment by dmitrylyzo** [comment]
Chrome 79 does not work
https://github.com/jellyfin/jellyfin-web/blob/de27ce0432e599a1dc24584e9f8cb0a784e74918/src/components/maintabsmanager.js#L190
`tabsElem` is `null`

File: src/scripts/site.js (hunk 1 above)
Comment (dmitrylyzo) [reply src/scripts/site.js:713]: This is from my patch. I tried to check module status:
* On PC and webOS 4, `fetch` is object
* On webOS 3, `fetch` is `undefined`, but it is works

**Comment by dmitrylyzo** [comment]
For Chrome 79, this works
```js
tabsElement = document.createElement('div', 'emby-tabs');
```

**Comment by JustAMan** [comment]
> For Chrome 79, this works
> 
> ```js
> tabsElement = document.createElement('div', 'emby-tabs');
> ```

But there is a `try..catch` block doing exactly this? Why doesn't it work? What does it say in the console?

Yeah, having a common place for both creating an element and for inserting it as a sole child (so as to replace `foo.innerHTML = bar`) would be good, I'd extract there if you can give me pointers where to put code.

**Comment by dmitrylyzo** [comment]
> Why doesn't it work? What does it say in the console?

It just creates a simple `div`.

File: src/scripts/site.js (hunk 1 above)
Comment (JustAMan) [reply src/scripts/site.js]: removed

**Comment by JustAMan** [comment]
@dmitrylyzo I think I found a better way, so it _should_ work now on all browsers.

File: src/components/maintabsmanager.js [hunk 2]
```diff
@@ -166,6 +166,7 @@ define(['dom', 'browser', 'events', 'emby-tabs', 'emby-button'], function (dom,
             }).join('') + '</div></div>';
 
             tabsContainerElem.innerHTML = tabsHtml;
+            CustomElements.upgradeSubtree(tabsContainerElem);
```
Comment (dmitrylyzo) [inline src/components/maintabsmanager.js:169]: to calm eslint
```suggestion
            window.CustomElements.upgradeSubtree(tabsContainerElem);
```

Comment (MrTimscampi) [reply src/components/maintabsmanager.js:169]: We can also add CustomElements to the globals list in .eslintrc.js, but using `window` is probably clearer.

**Pull request merged**